task2/src/App.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
btw, u use latest 17 React - no need to import it anymore)

Reply:
use jsx files

Reply:
FYI. To use .jsx transformation on babel 7, you must to include it to babel preset option: runtime: automatic.
On babel 8 it will be default value